apps/website/src/app/layout.tsx

```diff
@@ -48,7 +52,12 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.variable}>{children}</body>
+      <body className={inter.variable}>
+        <Header />
+        {children}
+        <Footer />
+        <NameGuardWrapper />
```

Comment:
I added this here so that the header/footer works across all pages as well as the `SearchModal`/`SettingsModal`